backend/queue/queue.go

```diff
@@ -171,24 +197,22 @@ func (q *Queue) swapLane(ctx context.Context, currentKey string, currentRevision
 // Enqueue adds a new value to the queue. It returns an error if the context is
 // canceled, the deadline exceeded, or if the client encounters an error.
 func (q *Queue) Enqueue(ctx context.Context, value string) error {
-	return q.tryPut(ctx, value)
-}
-
-func (q *Queue) tryPut(ctx context.Context, value string) error {
+	resp, err := q.cluster.MemberList(ctx)
+	if err != nil {
+		return fmt.Errorf("queue: couldn't enqueue item: %s", err)
+	}
+	if err := q.cleanup(ctx, resp.Members); err != nil {
```

Comment:
It's easier to prove correct this way. I'd prefer to keep doing it this way until we have a known performance issue. The API response time in testing was instant for me, and since this queue is only to be used for ad-hoc for now, I think it's probably OK.